markdown/3.0/en/ff-breadcrumb-trail.md

```diff
@@ -1,8 +1,8 @@
 ## Bare BCT
 
 By using a bare element, we tell the BCT to default everything. This means that the separator,
- which is displayed between the `ff-breadcrumb-trail-items`, defaults to '<' and the `ff-breadcrumb-trail-item` 
- innerHTML defaults to the {{description}} data binding
+ which is displayed between the `ff-breadcrumb-trail-items`, defaults to `<span data-separator> > </span>` and the `ff-breadcrumb-trail-item` 
```

Comment:
The change of _default separator template_ from `'<'` to `<span data-separator> > </span>` is to be discussed in related source code review

Reply:
I changed it back. The default separator is '>', not '<'.